src/main/java/com/github/ygimenez/method/Pages.java

```diff
@@ -1011,7 +1012,7 @@ public static void buttonize(@Nonnull Message msg, @Nonnull Map<Emoji, ThrowingC
 		if (useBtns) {
 			List<ActionRow> rows = new ArrayList<>();
 
-			List<Component> row = new ArrayList<>();
+			List<ActionComponent> row = new ArrayList<>();
```

Comment:
Would be better to refrain from using subclass types since the library might support Components other than Button futurely.